frontend/src/components/Footer.vue

```diff
@@ -5,116 +5,119 @@
+              <a style="padding-right:50px" v-for="url in csesocInternal" :key="url.name" :href="url.url">
+              {{ url.name }}
+              </a>
+            </div>
+            <v-row style="margin-top:25px" no-gutters>
+              <v-col
+                sm="5"
+              >
+                <h2>Social Media</h2>
+                <v-list-item class="anchor-items" v-for="url in media" :key="url.name">
+                  <v-list-item-content>
+                    <a :href="url.url">
+                      {{ url.name }}
+                    </a>
+                  </v-list-item-content>
+                </v-list-item>
+              </v-col>
+              <v-col
+                sm="5"
+              >
+                <h2>For your better future</h2>
+                <v-list-item class="anchor-items" v-for="url in csesocExternal" :key="url.name">
+                  <v-list-item-content>
+                    <a :href="url.url">
+                      {{ url.name }}
+                    </a>
+                  </v-list-item-content>
+                </v-list-item>
+              </v-col>
+            </v-row>
+          </div>
+        </v-col>
+      </v-row>
+    </v-container>
   </v-footer>
 </template>
 
 <script type="text/javascript">
-import APIClient from '../utils/APIClient';
-
 export default {
   name: 'Footer',
   data: () => ({
     // sponsors have name, logo, 'tier' value, and link
-    sponsors: []
-  }),
-  computed: {
-    // functions to determine sizing category of sponsor based on their value
-    tierOne() {
-      return this.sponsors.filter((x) => x.tier === 2);
-    },
-    tierTwo() {
-      return this.sponsors.filter((x) => x.tier === 1);
-    },
-    tierThree() {
-      return this.sponsors.filter((x) => x.tier === 0);
-    }
-  },
-  mounted() {
-    APIClient.fetchSponsors()
-      .then((responseJson) => {
-        this.sponsors = responseJson;
-      });
-  }
+    csesocInternal: [
+      { name: 'Home', url: '/' },
+      { name: 'About', url: '/#/about' },
+      { name: 'Resources', url: '/#/resources' },
+      { name: 'Sponsorships', url: '/#/sponsors' },
+      { name: 'Engage', url: '/#/engage' }
+    ],
+    csesocExternal: [
+      { name: 'CSESoc Media', url: 'https://media.csesoc.org.au/' },
+      { name: 'CSESoc Blog', url: '#' },
+      { name: 'CSESoc Compclub', url: '#' },
+      { name: 'UNSW CSE', url: 'https://www.engineering.unsw.edu.au/computer-science-engineering/' },
+      { name: 'UNSW Handbook', url: 'https://www.handbook.unsw.edu.au/' }
+    ],
+    media: [
+      { name: 'Facebook page', url: 'https://www.facebook.com/csesoc/' },
+      { name: 'Facebook Group', url: 'https://www.facebook.com/csesoc/' },
+      { name: 'Instagram', url: 'https://www.instagram.com/csesoc_unsw/' },
+      { name: 'Discord Community', url: 'https://forms.office.com/Pages/ResponsePage.aspx?id=pM_2PxXn20i44Qhnufn7o6ecLZTBorREjnXuTY-JfmBUMEpOMFBDTU1UWkhBWllWRTNPOVJFMUNCRi4u' },
```

Comment:
I like the way this has been written. I will uphold this standard for the rest of our landing page components. Thanks Jie.